browser/ui/views/brave_actions/brave_news_action_view.h

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_BROWSER_UI_VIEWS_BRAVE_ACTIONS_BRAVE_NEWS_ACTION_VIEW_H_
+#define BRAVE_BROWSER_UI_VIEWS_BRAVE_ACTIONS_BRAVE_NEWS_ACTION_VIEW_H_
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "base/memory/raw_ptr.h"
+#include "brave/browser/brave_news/brave_news_tab_helper.h"
+#include "brave/components/brave_today/browser/brave_news_controller.h"
+#include "brave/components/brave_today/browser/publishers_controller.h"
```

Comment:
nit: These can be deleted.

Reply:
:+1: sorry, I really struggle cleaning up imports. Is there any way to automate it?

Reply:
Well.. I don't much about that automation. maybe @mkarolin know?